src/Nancy.Swagger/Standalone.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Nancy.Swagger.Services;
+using Nancy.TinyIoc;
+using Swagger.ObjectModel;
+
+namespace Nancy.Swagger
+{
+    public static class Standalone
+    {
+        public static SwaggerRoot Generate(TinyIoCContainer container, NancyContext context)
```

Comment:
Taking a step back, that actually seems like a better solution. I kept reducing and reducing the amount of code involved in this PR. I'm closing this in favor of a Wiki page.